doc/release_notes_1.0.rst

```diff
@@ -0,0 +1,494 @@
+partition managing the device. It allows assigning I/O devices to a VM, 
+and extending the protection and isolation properties of VMs for 
+I/O operations.
+
+PIC/IOAPIC/MSI/MSI-X/PCI/LAPIC
+==============================
+ACRN hypervisor supports virtualized APIC-V, EPT, IOAPIC, and LAPIC functionality.
+
+One E2E binary to support all UEFI platform
+===========================================
+ACRN can support both Apollo Lake (APL) and Kaby Lake (KBL) NUCs.
+Instead of having separate builds, this release
+offers community developers a single end-to-end reference build that supports
+both UEFI hardware platforms, configured with a new board parameter.
+See :ref:`getting_started` for more information.
+
+Enable post-launched RTVM support for real-time UOS in ACRN
+===========================================================
+Initial patches to enable a User OS (UOS) running as a
+virtual machine (VM) with real-time characteristics,
+also called a “post-launched RTVM”. We’ve published a tutorial
+:ref:`rt_linux_setup`. More patches for ACRN real time support will continue.
+
+Enable cache QOS with CAT
+=========================
+Cache Allocation Technology (CAT) is available on Apollo Lake (APL) platforms,
+providing cache isolation between VMs mainly for real-time performance quality 
+of service (QoS). The CAT for a specific VM is normally set up at boot time per 
+the VM configuration determined at build time. For debugging and performance 
+tuning, the CAT can also be enabled and configured at runtime by writing proper 
+values to certain MSRs using the ``wrmsr`` command on ACRN shell.
+
+Enable QoS based on runC container
+==================================
+ACRN supports Device-Model QoS based on runC container to control the SOS 
+resources (CPU, Storage, MEM, NET) by modifying the runC configuration file.
+
+S5 support for RTVM
+===================
+ACRN supports a Real-time VM (RTVM) shutting itself down. A RTVM is a kind 
+of VM that the SOS can’t interfere with at runtime, and as such, only the 
+RTVM can power itself off internally. All power-off requests external to the 
+RTVM will be rejected to avoid any interference.
+
+Ethernet
+========
+ACRN hypervisor supports virtualized Ethernet functionality. The Ethernet 
+Mediator is executed in the Service OS and provides packet forwarding between 
+the physical networking devices (Ethernet, Wi-Fi, etc.) and virtual devices in
+the Guest VMs (also called “User OS”). The Virtual Ethernet device can be shared 
+by Linux, Android, and Service OS guests for regular (i.e. non-AVB) traffic.
+
+Storage (eMMC)
+==============
+ACRN hypervisor supports virtualized non-volatile R/W storage for the Service 
+OS and Guest OS instances, supporting VM private storage and/or storage shared 
+between Guest OS instances.
+
+USB (xDCI)
+==========
+ACRN hypervisor supports virtualized assignment of all USB xHCI and/or xDCI 
+controllers to a Guest OS from the platform.
```

Comment:
I don't quite understand this, is referring to passing an entire USB controller to a Guest (as a pass-through)?